setup.py

```diff
@@ -64,8 +65,8 @@ def load_requirements(*requirements_paths):
         'sample_xblocks.filethumbs',
         'workbench',
     ],
-    install_requires=load_requirements('requirements/base.txt'),
-    tests_require=load_requirements('requirements/test.txt'),
+    install_requires=load_requirements('{}/requirements/base.txt'.format(os.getcwd())),
+    tests_require=load_requirements('{}/requirements/test.txt'.format(os.getcwd())),
```

Comment:
Shouldn't the new Django requirements file be added here to preserve the fact that the package depends on it?  tox should process the explicit dependencies from `deps` before it looks at setup.py, and the file uses a range of versions compatible with all the tox configurations.

Reply:
Yes - it should. I'll add it.